js-setup/templates/bootstrapper.html

```diff
@@ -54,15 +54,15 @@
 				'Array.prototype.find|always|gated',
 				'Array.prototype.findIndex|always|gated',
 				'IntersectionObserver'
-			].join(',') + '&excludes=Symbol,Symbol.iterator,Symbol.species,Map,Set');
+			].join(',') + '&excludes=Symbol,Symbol.iterator,Symbol.species,Map,Set' + '&flags=gated&unknown=polyfill');
 		} else {
 			{{#if hasMainCoreJs}}
 			ftNextLoadScript('{{#hashedAsset}}main-core.js{{/hashedAsset}}');
 			{{/if}}
 			ftNextLoadScript(polyfillUrl + '?callback=' + polyfillCallbackName + '&features=' + [
 				'default',
 				'HTMLPictureElement|always|gated'
-			].join(','));
+			].join(',') + '&flags=gated&unknown=polyfill');
```

Comment:
The querystring parameters and values you've added will ensure that useragents unknown to the polyfill-service will be served all polyfills you have asked for and will wrap each polyfill in a feature detection as to not polyfill anything which the useragent has already implemented.

If your issue is with FB in-app browser, we have a PR to add detection for it -- https://github.com/Financial-Times/polyfill-service/pull/775

I'm not sure how this application is setup, it looks to be calling the polyfill-service twice? Is that correct?

/cc @triblondon as he is looking at FB in-app browser.